nkf.gemspec

```
@@ -28,8 +28,16 @@ Gem::Specification.new do |spec|
   spec.files         = Dir.chdir(File.expand_path('..', __FILE__)) do
     `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
   end
+
+  if Gem::Platform === spec.platform and spec.platform =~ 'java' or RUBY_ENGINE == 'jruby'
+    spec.platform = 'java'
+    spec.licenses      = ["EPL-2.0", "LGPL-2.1"]
```

```suggestion
    spec.licenses      += ["EPL-2.0", "LGPL-2.1"]
```

Sorry to nit-picking. gemspec or some files are distributed by Ruby or BSD-2-Clause.

---

Oops, that was supposed to be +=. Will fix.

---

Fixed in latest push.